### PR TITLE
[PM-2054] Updated Password Generator History to use Component Library on Web

### PR DESCRIPTION
**Pull request by aj-rosado**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Update the Password generator history to use Component Library on Web client

## Code changes

Update to use Component Library 

## Screenshots

**Before:**
password-history-old-empty
password-history-old-copied
password-history-old-layout

**After:**
password-history-new-empty
password-history-new-copied
password-history-new-layout

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: apps/web/src/app/tools/generator.component.ts [hunk 1]
```diff
@@ -28,6 +29,7 @@ export class GeneratorComponent extends BaseGeneratorComponent {
     i18nService: I18nService,
     logService: LogService,
     route: ActivatedRoute,
+    private dialogService: DialogServiceAbstraction,
     private modalService: ModalService
```
Comment (djsmith85) [inline apps/web/src/app/tools/generator.component.ts:33]: As the modalService is replaced by the DialogService, we can remove it from the ctor and the import

**Review by djsmith85 — CHANGES_REQUESTED** [review]
@aj-rosado: Thanks for tackling this.

Just some minor clean-up.

There's still a Lint issue, execute `npm ci`, followed by `npm run prettier`. This should resolve the issue on `apps/web/src/app/tools/password-generator-history.component.html`

**Review by djsmith85 — DISMISSED** [review]
Changes are looking good

**Review by Hinton — COMMENTED** [review]
Looks good, some minor feedback, nothing blocking but I think we can clean it up slightly further. Would appreciate @djsmith85 thoughts on the table use as well.

File: apps/web/src/app/tools/password-generator-history.component.html [hunk 2]
```diff
@@ -1,59 +1,50 @@
...
+            bitIconButton="bwi-clone"
+            (click)="copy(h.password)"
+            [attr.title]="'copyPassword' | i18n"
+            [attr.aria-label]="'copyPassword' | i18n"
+            [appA11yTitle]="'copyPassword' | i18n"
+          ></button>
         </div>
-      </div>
+      </li>
+    </ul>
```
[the remark this answers is not in the document]
Comment (aj-rosado) [reply apps/web/src/app/tools/password-generator-history.component.html]: Changed and updated screenshots

**Review by Hinton — APPROVED** [review]
Looks great thanks!

**Review by djsmith85 — APPROVED** [review]

**Pull request merged**